bindings/rust/aws-lc-sys-template/Cargo.toml

```diff
@@ -25,9 +25,11 @@ include = [
 
 [features]
 asan = []
+bindgen = ["dep:bindgen"]
```

Comment:
I believe the [`dep:` syntax](https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies) is only available in Rust 1.60 or later.  We currently claim support for 1.57.